GUI/methods.py

```diff
@@ -206,7 +206,7 @@ def addLabel(self):
 			fd.write(newLabel)
 			fd.close()
 		else:
-			fd.open('./.data/' + self.CLASS_NAME + '/labels.txt', 'a+')
+			fd = open('labels.txt', 'a+')
```

Comment:
Similar comment: this assumes we want the file adjacent to this file?

Reply:
Should be . method instead of = ?

Reply:
Nvm this should probably be =